test/test_smoke.py

```diff
@@ -91,9 +91,14 @@ def test_smoke(output_path, config_json):
     else:
         print("WARNING: selinux not enabled, cannot check for denials")
 
-    with VM(generated_img) as test_vm:
-        exit_status, _ = test_vm.run("true", user="test", password="password")
-        assert exit_status == 0
-        exit_status, output = test_vm.run("echo hello", user="test", password="password")
-        assert exit_status == 0
-        assert "hello" in output
+    # building an ELN image needs x86_64-v3 to work, we use avx2 as a proxy
+    # to detect if we have x86-64-v3 (not perfect but should be good enough)
+    if " avx2 " not in pathlib.Path("/proc/cpuinfo").read_text():
+        print("WARNING: no x86_64-v3 cpu detected, skipping VM boot test")
```

Comment:
Fwiw, this is ugly, I will look into making this a proper "pytest.mark.skipif" somehow

Reply:
Either way is fine with me, I think it's okay to land and unblock PRs but I also really want to push a PR before my EOD that untangles this mess a bit more

Reply:
But might be best as a followup just to have an easier review/discussion